### ISSUE TITLE
SlevomatCodingStandard.Variables.UselessVariable.UselessVariable - false positive

### ISSUE THREAD

**Issue by MartkCz** [slevomat/coding-standard#1144]
Useless variable $item

```php
TypeAssertion::assertType($item = $this->context[$key], $type);

return $item;
```

ok:

```php
$item = $this->context[$key];
TypeAssertion::assertType($item, $type);

return $item;
```

**Comment by kukulich** [comment]
Thank you for your report but I will not fix it.
